Cult of St. Credit, a new Chaplain sect and money sink.

tarlockgod opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

I added a new sect for the chaplain, the Cult of St. Credit.
You can only pay the cult in holochips loaded with credits, everything else is heresy!
Joint today, become rich!

I also dont expect this to be merged(at least for now), its more of a first taste of byond coding for me. If the idea is liked though(besides the one guy in discord who told me to merge) i might expand it.

**EDIT: Besides some small fixes if any problems are found or pointed out, this PR is now feature complete**

## Why It's Good For The Game

Its gives the chap a bit more content, and a nice money sink for the assistants of the station, since the price is a money golem!
The golem rite might be a bit OP, but i think its fine for such a money sink.

## Changelog
:cl:
add: new sect for the chap to play with(also money sink, since you get favour with credits(holochips to be exact))
add: rite to summon some capitalist clothing
add: added bless mechanic, works the same as normal, but you pay 10 credits per hit. so run away or be forced to donate
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

tarlockgod committed:
money machine go BRRRRRRRRRRRRRRRRRRRRRRRRRRR

tarlockgod committed:
added second rite, figured out the location thing

tarlockgod committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into sect2

LemonInTheDark commented:
need a newline there 
![image](https://user-images.githubusercontent.com/58055496/87087699-6cca6b80-c1e8-11ea-87a8-563bb7082352.png)

Don't think this needs to be a draft, unless things are totally unfinished

tarlockgod committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into sect2

tarlockgod commented:
I made it a draft becasue i might expand it(Arcane also told me in my first PR to make it a draft instead).
Also, do you really think its good to merge?

tarlockgod committed:
newline fixed(i hope)

tarlockgod commented:
Ah good, newline fixed
If nobody complains i am gonna take it off draft tomorrow. Might add some new things if i get a good idea.

Fikou commented on code/modules/religion/religion_sects.dm:
```diff
@@ -190,3 +190,23 @@
+	desc = "A cult oriented around money."
+	convert_opener = "If you always donate your money and dont violate the NAP, you too might one day achieve the top 0.0000001%!<br>(Only Holochips accepted, for more questions reach out to our legal team!)"
+	alignment = ALIGNMENT_EVIL
+	desired_items = list(/obj/item/holochip)
```
why doesnt it accept space cash 😏

Fikou commented:
you might want to multiply that cash needed for the golem rite by 10, its a really powerful thing

ATH1909 commented:
eh, I feel like this might ruin the "special"-ness of capitalist golems

make how much the bible effect heals someone dependent on how much cash they have in their ID (with a cap, of course, and maybe with a certain percent of that cash being automatically "donated" to the church on a successful heal)

tarlockgod commented:
Good idea, i will have to see if i can do that

tarlockgod commented:
Oh god economy is a bit too bigbrained for my right now, i am gonna try to finish it tomorrow.
But the chap going around extorting money for healing seems a bit OP unless i oull the numbers really down.

tarlockgod committed:
Upped the cost for the money golem to 100k

optimumtact commented:
![image](https://user-images.githubusercontent.com/2544323/87105192-5d155a80-c2ae-11ea-9499-41a717c21dd8.png)

tarlockgod commented:
I know oranges. Any thoughts on how to improve it? I can add more rites, or give them the bless-for-money thing that was suggested before. Its never gonna be merged anyway, so shoot.

Firecage commented:
Honestly, the Golem thing is a bit to much. Considering that either Capitalist or Communist golems, in effect, makes something like Romerol look like the common cold compared to the Plague. They are, in effect, round enders of the worst kind due to both the conversion thing and the knock spell.

tarlockgod commented:
That could indeed work, a, "church approved" version so to speak.
I am also gonna lower its cost then, since its not that extreme anymore.
With that and the sect bless i am working on, i would say the PR is in a good place.

tarlockgod committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into sect2

tarlockgod committed:
created the Churchgoing Capitalist Golem, a church approved downgrade
also lowered the cost to become one to 10k favour since its not the roundender type anymore

tarlockgod commented:
Since Finals are near and i dont wanna spend more time figuring out how to deduct money from peoples IDs upon being blessed, i am gonna let this PR leave draft. If it, somehow, gets merged i will add the bless after finals are done.

Tlaltecuhtli commented:
.adjust_money(-value)  removes money, paste it in github to find similar places where its used so you know how the proc works,

and bless is triggered in /datum/religion_sect/proc/sect_bless(mob/living/L, mob/living/user)

tarlockgod committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into sect2

tarlockgod committed:
added bless mechanic, works the same as normal, but you pay 10 credits per hit. so run away or be forced to donate

tarlockgod commented:
Thanks tlat, found the soulution to the payment problem thanks to your mention of the search function(didnt know it existed).

Also, this PR is now complete. Big thanks you to the people who commented here, you helped alot.

Even if it doesnt get merged, making it was a very informative journey into SS13 code.

tarlockgod committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into sect2

# Conflicts:
#	code/modules/mob/living/carbon/human/species_types/golems.dm
#	tgstation.dme

ExcessiveUseOfCobblestone commented:
blursed meme sect

optimumtact commented:
You are responsible for this cobby

Iamgoofball commented:
NUMBER
GO
UP

ExcessiveUseOfCobblestone commented:
I dont want this in normal play as its a bit too memey for my tastes.

The options are TM for lols, configure it as admin-only, or close.

tarlockgod commented:
Ye, its very memey. Admin only would fit for it. I could also downgrade the memefactor a bit since my finals are nearly done and i have time again.

Iamgoofball commented:
Just ditch the golem bits and it'd be fine?

Scummy rich people cult is like 90% of religion, the fact we didn't have a cult for it already is absurd.

tarlockgod committed:
removed golem type and golem ritual

tarlockgod commented:
Removed the golem, played around with some other rituals, didnt come up with anything good. If it doesnt get accepted i am gonna close it.
